### PR TITLE
Prepare v1.7.0

### PR DESCRIPTION
**Pull request by ShuCh3n**

### CODE REVIEW

File: tests/Buckaroo/Payments/NoServiceSpecifiedPaymentTest.php [hunk 1]
```diff
@@ -1,41 +1,43 @@
...
-            'amountDebit' => 10,
-            'invoice' => uniqid(),
-            'servicesSelectableByClient' => 'ideal,bancontactmrcash,paypal',
-            'servicesExcludedForClient' => 'ideal',
-            'continueOnIncomplete' => '1',
-        ]);
-        $this->assertTrue($response->isWaitingOnUserInput());
-    }
-}
+<<<<<<< HEAD
```
Comment (vegimcarkaxhija) [inline tests/Buckaroo/Payments/NoServiceSpecifiedPaymentTest.php:1]: what happened here

File: src/Models/Payload/Payload.php [hunk 2]
```diff
@@ -22,8 +22,12 @@
 
 use Buckaroo\Models\AdditionalParameters;
 use Buckaroo\Models\ClientIP;
+use Buckaroo\Models\CustomParameters;
 use Buckaroo\Models\Model;
 
+/**
+ *
```
Comment (vegimcarkaxhija) [inline src/Models/Payload/Payload.php:29]: remove comment

File: src/Config/Config.php [hunk 3]
```diff
@@ -26,9 +26,18 @@
 use Buckaroo\Handlers\Logging\Loggable;
 use Buckaroo\Handlers\Logging\Subject;
 
+/**
+ *
```
Comment (vegimcarkaxhija) [inline src/Config/Config.php:30]: remove comment

File: tests/Buckaroo/Payments/NoServiceSpecifiedPaymentTest.php (hunk 1 above)
Comment (ShuCh3n) [reply tests/Buckaroo/Payments/NoServiceSpecifiedPaymentTest.php:1]: Part of merge conflict, fixed

File: src/Models/Payload/Payload.php (hunk 2 above)
Comment (ShuCh3n) [reply src/Models/Payload/Payload.php:29]: Removed for now, but this doesn't really matter. It's auto-generated by IDE

**Review by vegimcarkaxhija — APPROVED** [review]

**Pull request merged**